services/ui-src/package.json

```diff
@@ -39,15 +40,14 @@
     "eslint-loader": "3.0.3",
     "eslint-plugin-flowtype": "4.6.0",
     "eslint-plugin-import": "2.20.1",
-    "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-react": "7.19.0",
     "eslint-plugin-react-hooks": "^1.6.1",
     "file-loader": "4.3.0",
     "file-saver": "^2.0.5",
     "fs-extra": "^8.1.0",
     "html-webpack-plugin": "^4.5.2",
     "html2canvas": "^1.0.0-rc.7",
-    "identity-obj-proxy": "3.0.0",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
left this in the build dependencies because I wasn't exactly sure where it should end up. I think it's only used in jest